packages/an-anime-game-launcher-bin/an-anime-game-launcher-bin.pacscript

```diff
@@ -1,9 +1,9 @@
 name=an-anime-game-launcher-bin
```

Comment:
```suggestion
# Detect if the OS is Debian and error out
if [[ -f /etc/debian_version ]]; then
    fancy_message error "This script doesn't work on Debian"
    cleanup # Trigger pacstall cleanups
    exit 1
fi

name=an-anime-game-launcher-bin
```

Error out if the OS is detected as Debian.

We'll add a variable (or an array) in the future where you can specify what OSes the package doesn't work on. But for now you'll have to manually program it and error out.